modules/common/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@erento/nestjs-common",
+  "version": "1.0.0",
+  "description": "NestJS module with some shared utilities",
+  "main": "dist/public_api.js",
+  "types": "dist/public_api.d.ts",
+  "scripts": {
+    "build": "tsc -p tsconfig.json",
+    "lint": "tslint -c tslint.json --fix -e \"**/node_modules/**\" -p ./tsconfig.json",
+    "lint-staged": "lint-staged",
+    "prebuild": "rm -rf dist && npm run lint && npm run test",
+    "test": "jest"
```

Comment:
you need to also make it running the test, see more here:
https://github.com/erento/nestjs-modules/blob/master/Jenkinsfile